source/mod-panel

```diff
@@ -456,18 +458,21 @@ if __name__ == '__main__':
     # --------------------------------------------------------------------------------------------------------
     # Set-up workdir environment as LV2_PATH, if relevant
 
-    if os.getenv("LV2_PATH") is None:
+    lv2_path = os.getenv("LV2_PATH")
+    if lv2_path is None:
```

Comment:
lv2_path and LV2_PATH point to the same thing here, best to just use LV2_PATH as single var